docs/best-practices/ci-pipeline-image-predownload.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Cloud-Native Devops Best Practices(1) - Continuous Integration (CI) + OpenKruise Image Pre-download
+---
+
+## What is Devops？
+DevOps is the merging of the three domains of Development, QA, and Operations. DevOps is an idea, a set of best practices, and a culture.
+DevOps is an extension of CI/CD, and CI/CD is the core foundation of DevOps. Without CI/CD automation tools and processes, DevOps is meaningless.
+
+![ci/cd pipeline](/img/docs/best-practices/ci_cd_pipeline.png)
+
+## Continuous Integration (CI) + OpenKruise Image Pre-download
+### Core Concepts
+- **Continuous Integration(CI)** is a hands-on way to bring integration forward to the early stages of the development cycle, allowing builds, tests and integration of code to happen more often and repeatedly.
+- **[Image Pre-download](https://openkruise.io/docs/user-manuals/imagepulljob/)** is provided by OpenKruise to pull application images to specific Node nodes in advance of application deployment, which in turn can greatly improve the efficiency of application deployment.
+
+### Architecture
+![ci+image predownload](/img/docs/best-practices/imagepulljob.png)
+
+### User Story
+- **Long-term pre-download operation and maintenance containers, base images, such as: istio envoy, logtail log collection container.**
```

Comment:
operation and maintenance containers -> common sidecar images?
 logtail log collection container -> log collection containers